docker/orthanc/AmazonLinux2Dockerfile

```diff
@@ -489,16 +455,12 @@ COPY *.lua /lua-scripts/
 COPY test-aliveness.py /probes/
 
 
-# cleanup unnecessary packages that can trigger errors during security scan
-RUN apt purge --assume-yes build-essential perl bzip2 gnupg xdg-user-dirs && apt --assume-yes autoremove
-
-
-# always create an 'orthanc' group (gid=999) and an orthanc user (uid=999)
+# always create an 'orthanc' group (gid=1000) and an orthanc user (uid=1000)
```

Comment:
Any specific reason we change the gid and uid from 999 to 1000?

Reply:
999 was already used by something else (i don't remember what off the top of my head), so i chose to match [this](https://github.com/uc-cdis/base-images/blob/debadb5/python-nginx/3.9/Dockerfile#L33-L35)